Translators/WZDx/snmp_operations.py

```diff
@@ -1,19 +1,23 @@
-import subprocess
-import logging
+import requests
+import json
 
 def clear_index(rsu):
-    '''Clears the specific RSU index'''
-    cmd = ''
-    index = rsu["rsuIndex"]
-    if rsu["snmpProtocol"] == "NTCIP1218":
-        cmd = f'snmpset -v 3 -u {rsu["rsuUsername"]} -a SHA -A {rsu["rsuPassword"]} -x AES -X {rsu["rsuPassword"]} -l authpriv {rsu["rsuTarget"]} 1.3.6.1.4.1.1206.4.2.18.3.2.1.9.{index} i 6'
+    deleteUrl = 'http://localhost:8080/tim'
```

Comment:
Let's abstract this URL out to an environment var.

Reply:
Updated to use environment var